pudb/theme.py

```diff
@@ -469,57 +469,116 @@ def link(child: str, parent: str):
-            "group head": (add_setting("black", "bold"), "light gray"),
-            "warning": (add_setting("light red", "bold"), "black"),
-            "header warning": (add_setting("light red", "bold"), "light gray"),
-            "dialog title": ("black", "dark green"),
-            "fixed value": ("white", "dark gray"),
+            "input": ("black", "dark cyan"),
+            "warning": (add_setting("white", "bold"), "dark red"),
+            "search box": ("white", "black"),
+            "dialog title": (add_setting("white", "bold"), "dark cyan"),
+            "group head": (add_setting("dark blue", "bold"), "light gray"),
+            "button": (add_setting("white", "bold"), "dark cyan"),
+            "focused sidebar": ("black", "white"),
+            "focused button": ("light cyan", "black"),
+            "value": (add_setting("yellow", "bold"), "dark blue"),
+            "fixed value": ("light gray", "dark blue"),
             # }}}
             # {{{ source view
-            "current source": ("black", "dark cyan"),
-            "breakpoint source": ("white", "dark gray"),
-            "line number": (add_setting("dark gray", "bold"), "black"),
-            "breakpoint marker": (
-                add_setting("light red", "bold"), "black"),
-            "current line marker": ("dark cyan", "black"),
+            "source": ("dark green", "black"),
+            "highlighted source": ("black", "dark green"),
+            "current source": ("black", "brown"),
+            "current focused source": (add_setting("yellow", "bold"), "dark blue"),
+            "focused source": ("white", "dark blue"),
+            "breakpoint source": (add_setting("yellow", "bold"), "dark red"),
+            "current breakpoint source": ("black", "dark red"),
+
+            "line number": ("light gray", "black"),
+            "current line marker": ("dark red", "black"),
+            "breakpoint marker": ("dark red", "black"),
             # }}}
             # {{{ sidebar
-            "sidebar two": ("dark green", "black"),
-            "sidebar three": ("yellow", "black"),
-            "focused sidebar three": ("dark gray", "light green"),
+            "sidebar two": ("light blue", "black"),
+            "sidebar three": ("light cyan", "black"),
             # }}}
             # {{{ variables view
-            "return label": ("black", "dark green"),
-            "focused return label": ("black", "dark green"),
+            "var label": ("light blue", "black"),
+            "var value": ("white", "black"),
+            "variable separator": ("dark cyan", "light gray"),
+
+            "focused var label": ("white", "dark blue"),
+            "focused var value": ("white", "dark blue"),
+
+            "highlighted var label": ("black", "dark green"),
+            "highlighted var value": ("black", "dark green"),
+            "focused highlighted var label": ("black", "light green"),
+            "focused highlighted var value": ("black", "light green"),
```

Comment:
```suggestion
```
For consistency, I'd say let these fallback or else use the dark blue background.